.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.3.3/apache-maven-3.3.3-bin.zip
```

Comment:
That's my bad I just copied over `.mvn` from my JRubyArt project (which includes maven-wrapper)